picard/item.py

```diff
@@ -164,9 +167,13 @@ def __init__(self):
         super().__init__()
         self.metadata = Metadata()
         self.orig_metadata = Metadata()
-        self.update_metadata_images_enabled = True
+        self.enable_update_metadata_images(True)
         self.update_children_metadata_attrs = {}
         self.iter_children_items_metadata_ignore_attrs = {}
+        self.suspend_metadata_images_update = IgnoreUpdatesContext(
+            on_first_enter=partial(self.enable_update_metadata_images, False),
+            on_last_exit=partial(self.enable_update_metadata_images, True),
+        )
```

Comment:
We don't need to function. Instead we could do directly

```IgnoreUpdatesContext
self.update_metadata_images_enabled = IgnoreUpdatesContext()
```

`IgnoreUpdatesContext` is designed to be treated as a boolean to allow this kind of usage. So you can still do:

```python
if self.update_metadata_images_enabled:
   ...
```

Reply:
Sorry, it is the other way around. We would need to call it `suspend_metadata_images_update` like you did, so we would need to turn around all current if checks (`if not self.suspend_metadata_images_update`  instead of `if self.update_metadata_images_enabled`).

But I still think we should do it :)

Reply:
It should be done in latest iteration, please check.